tests/test_upload.py

```diff
@@ -18,6 +18,10 @@
 from twine.commands import upload
 from twine import package, cli, exceptions
 import twine
+from twine.exceptions import (
+    InvalidPyPIUploadURL,
```

Comment:
We only ever import modules throughout the project, let's hold to that in the tests too please

Reply:
`hacking` used to do this too but it also introduced lots of other checks that didn't seem worth the effort to ignore.